controllers/actionController.js

```diff
@@ -368,3 +369,26 @@ exports.sendMessage = asyncErrorHandler(async (req, res, next) => {
         return res.status(500).json({message: "Internal Server Error"})
     }
 })
+
+// FEEDBACK SECTION
+exports.addFeedback = asyncErrorHandler(async (req, res, next) => {
+    const userId = req.user._id;
+    const { feedbackMessage } = req.body;
+
+    if (!(feedbackMessage)) {
+        return res
+            .status(403)
+            .send({message: 'Feedback message cant be empty'});
```

Comment:
@Kelvin-K can you use json instead of send as it is used in the app in JSON format !

Reply:
Done